typings/karma-fixtures.d.ts

```diff
@@ -208,7 +208,6 @@ interface RuleTemplates {
   };
   TextInput: {
     Equals: IRuleDescription;
```

Comment:
@nishantwrp why do these still have "I" at the beginning? /cc @vojtechjelinek

Reply:
Hmm, I'm a little confused here. I changed the name of the actual interface `IRuleDescription` to `RuleDescription`. But still typescript didn't throw any error here. Maybe it doesn't for `.d.ts` files. Also, I checked other `.d.ts` this seems the only file to have the `I` prefix. @shavavo can you please remove the `I` prefix from this file?

Reply:
@nishantwrp could you please dig into why typescript didn't throw an error? Our whole strategy here depends on having checks that actually catch errors when they occur. If it doesn't throw an error then we'll need to figure out how to throw one ourselves and implement that.

Reply:
This issue (https://github.com/microsoft/TypeScript/issues/36971) explains why that happens.

Reply:
@nishantwrp sure, I've removed all I prefixes.

Reply:
@nishantwrp we still need to do something about it, right? (possibly on our end)

Otherwise why have these files at all?

Reply:
Filed #10219